src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -61,7 +63,8 @@
                 (when-let [selected-entity-type (or (:selected-entity-type @state) (:initial-entity-type @state) (first (:entity-types @state)))]
                   [:a {:style {:textDecoration "none" :float "left" :margin "7px 0 0 1em"}
                        :href (str "/service/api/workspaces/" (:namespace (:workspace-id props)) "/"
-                               (:name (:workspace-id props)) "/" selected-entity-type "/tsv")
+                               (:name (:workspace-id props)) "/entities/" selected-entity-type "/tsv")
+                       :onClick (fn [] (.set goog.net.cookies "FCtoken" @access-token 300))
```

Comment:
It is so awesome that you did this instead of my custom cookies stuff. I always forget that this entire library is available to us.